bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -286,63 +343,6 @@ spec:
       - name: ovn-cert
         secret:
           secretName: ovn-cert
```

Comment:
I believe the `volume` entry in both `ovnkube-node.yaml` and `ovnkube-master.yaml` needs to be made `optional: true` (as it is in `bindata/network/openshift-sdn/sdn.yaml`), because the secret can't be created until after the network is already up, but if the volume is required then the network won't come up without the secret having already been created.

Reply:
Tests are failing precisely because of this